mitxgraders/helpers/calc/calc.py

```diff
@@ -591,7 +630,7 @@ def handle_node(node):
             if any(np.any(np.isnan(r)) for r in as_list):
                 return float('nan')
 
-            return result
+            return cast_np_numeric_as_builtin(result, map_across_lists=False)
```

Comment:
Did you mean to set this True?

Reply:
Oops, that was a relic from trying to find a test that would break when `False`. Fixed.